node/Makefile

```diff
@@ -437,7 +435,7 @@ st: st-checks $(REMOTE_DEPS) image dist/calicoctl busybox.tar calico-node.tar wo
 ci: static-checks ut image image-windows st
 
 ## Deploys images to registry
-cd: cd-common release-windows
```

Comment:
We have a dedicate job to `release-windows` so no need to list it as a dependency here. It also fails on arm64 runner.